src/quickstart/provider/aws/boot.md

```diff
@@ -1,263 +1,364 @@
 
-## Pre-requisites
 
-1. You need an active AWS account. Visit [AWS portal](http://aws.amazon.com) to sign up
-2. You need AWS API keys with full access
-3. Install the AWS cli tools, you can find instructions for your platform at [AWS Command Line Interface Site](https://aws.amazon.com/cli/)
+## Installing kops
 
-To verify that your CLI is configured properly, run `aws ec2 describe-regions`:
+Download the [latest](https://github.com/kubernetes/kops/releases/latest) version of kops
 
+
+#### macOS
+
+```bash
+curl -sSL https://github.com/kubernetes/kops/releases/download/1.5.1/kops-darwin-amd64 -O
+chmod +x kops-darwin-amd64
+sudo mv kops-darwin-amd64 /usr/local/bin
 ```
-$ aws ec2 describe-regions --output=text
-REGIONS	ec2.eu-west-1.amazonaws.com	eu-west-1
-REGIONS	ec2.ap-southeast-1.amazonaws.com	ap-southeast-1
-REGIONS	ec2.ap-southeast-2.amazonaws.com	ap-southeast-2
-REGIONS	ec2.eu-central-1.amazonaws.com	eu-central-1
-REGIONS	ec2.ap-northeast-2.amazonaws.com	ap-northeast-2
-REGIONS	ec2.ap-northeast-1.amazonaws.com	ap-northeast-1
-REGIONS	ec2.us-east-1.amazonaws.com	us-east-1
-REGIONS	ec2.sa-east-1.amazonaws.com	sa-east-1
-REGIONS	ec2.us-west-1.amazonaws.com	us-west-1
-REGIONS	ec2.us-west-2.amazonaws.com	us-west-2
+
+
+#### linux
+
+```bash
+curl -sSL https://github.com/kubernetes/kops/releases/download/1.5.1/kops-linux-amd64 -O
+chmod +x kops-darwin-amd64
+sudo mv kops-darwin-amd64 /usr/local/bin
 ```
 
-## Download and Unpack Kubernetes
+For more information see the official [kops installation guide](https://github.com/kubernetes/kops/blob/master/docs/aws.md)
+
+## Validate kops is installed
+
+```
+kops version
+Version 1.5.1
+```
 
-First, make a directory to hold the Kubernetes release files:
+## Install kubectl if you haven't done so yet
 
 ```
-$ mkdir my-first-cluster
-$ cd my-first-cluster
+curl -LO https://storage.googleapis.com/kubernetes-release/release/$(curl -s https://storage.googleapis.com/kubernetes-release/release/stable.txt)/bin/darwin/amd64/kubectl
+chmod +x kubectl
+sudo mv kubectl /usr/local/bin
 ```
 
-See [Kubernetes Versions](https://deis.com/docs/workflow/installing-workflow/system-requirements/#kubernetes-versions) under System Requirements and download a Kubernetes release that is compatible with Deis Workflow, and extract the archive on your machine.
 
-This archive has everything that you need to launch Kubernetes. It's a fairly large archive, so it may take some time to download:
+## Setup your AWS account
+
+#### Setup an IAM user for kops
+
+In order to build clusters within AWS we'll create a dedicated IAM user for
+`kops`.  This user requires API credentials in order to use `kops`.  Create
+the user, and credentials, using the [AWS console](http://docs.aws.amazon.com/AWSSimpleQueueService/latest/SQSGettingStartedGuide/AWSCredentials.html).
+
+The `kops` user will require the following IAM permissions to function properly:
```

Comment:
Can we go into just a bit of detail here so the user knows why we need the following IAM permissions to have full access?